src/Chapter1.hs

```diff
@@ -653,8 +677,13 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+isDigit :: Int -> Bool
+isDigit n = n >= 0 && n <= 9
 
+firstDigit :: Int -> Int
+firstDigit n
+    | isDigit n = n
+    | otherwise = firstDigit (div n 10)
```

Comment:
That is nice 💯 
Although, you might want to use `abs n` here instead of just `n` to handle negative cases 🙂